clients/src/test/java/org/apache/kafka/common/network/ChannelBuildersTest.java

```diff
@@ -38,7 +39,6 @@
 public class ChannelBuildersTest {
 
     @Test
-    @SuppressWarnings("deprecation")
```

Comment:
Why did we remove this?

Reply:
This test does not use any deprecated classes, so it's unecessary

Reply:
I see, `OldPrincipalBuilder` extends the deprecated class but it's not deprecated itself.